users/serializers.py

```diff
@@ -1,11 +1,12 @@
 from rest_framework import serializers
-from rest_framework.response import Response
 from django.contrib.auth.models import User
 
+
 class UserListSerializer(serializers.ModelSerializer):
     class Meta:
         model = User
-        fields = ('email', 'username', )
+        fields = ('email', 'username',)
```

Comment:
You open user email's to the world. Generally speaking that's something you shouldn't do.